src/org/mockito/internal/verification/AtLeast.java

```diff
@@ -58,4 +58,12 @@ public void verifyInOrder(VerificationDataInOrder data) {
     public String toString() {
         return "Wanted invocations count: at least " + wantedCount;
     }
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
```

Comment:
unnecessary javadoc as well (all the `@inheritDoc`), it's eclipse that generates those stubs right ;)